vg/vgimg/vgimg.go

```diff
@@ -53,8 +54,8 @@ const (
 )
 
 // New returns a new image canvas.
-func New(w, h vg.Length) *Canvas {
-	return NewWith(UseWH(w, h))
+func New(w, h vg.Length, c color.Color) *Canvas {
```

Comment:
I'm not sure this is necessary. @sbinet?

Reply:
I don't think this requires an api change.

I think the New function arguments should stay the same, but the body should be:

```
Return NewWith(UseWH(w,h), UseColor(color.White))
```

Then there can be:

``` go
func UseColor(c color.Color) option {
return func(c *Canvas) uint32 {
		draw.Draw(c.img, c.img.Bounds(), image.White, image.ZP, draw.Src)
		return setsColor
	}
}
```

Then we just need to add a setsColor constant and delete the existing draw.Draw line from NewWith.

I'm on a phone now but can clarify later if necessary.

Reply:
...and `UseImg` will need to return `setsSize&setsColor`

Reply:
I agree with @ctessum.

Reply:
@MiguelPires Please address this.

Reply:
Please remove this addition.